wgengine/router/router_linux.go

```diff
@@ -46,6 +46,11 @@ const (
 	tailscaleBypassMark = "0x80000"
 )
 
+// tailscaleRouteTable is the routing table number for Tailscale
+// network routes. See addIPRules for the detailed policy routing
+// logic that ends up doing lookups within that table.
+const tailscaleRouteTable = "52"
```

Comment:
Note the TS mnemonic here? And mention that it has to be less than 256 for Busybox?

Reply:
Done.